source/blog/2018-10-12-the-ember-times-issue-68.md

```diff
@@ -27,21 +27,20 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
-
+## [Find Ways to Help Out with the Ember Help Wanted App ✨](https://help-wanted.emberjs.com)
 
----
+Too much spare time at hand and don't know what to do? Wanna **contribute to open-source**? Then check out the sparkling ✨✨ [new **Ember Help Wanted App**](https://help-wanted.emberjs.com) ✨✨!!
 
-## [SECTION TITLE](#section-url)
+The Ember Help Wanted App lists all current open issues across Ember repositories, like Ember CLI, Ember Data, the Website and more which could use some support. The website is also a great resource if you're still looking to get your [**Hacktoberfest**](https://www.emberjs.com/blog/2018/10/05/the-ember-times-issue-67.html#toc_a-href-https-hacktoberfest-digitalocean-com-hacktoberfest-a) contributions in, so be sure to [check it out](https://help-wanted.emberjs.com).
```

Comment:
For `like Ember CLI, Ember Data, the Website` can say `like Ember CLI, Ember Data, the EmberJS Website` or something similar since I don't think most would put it together

Reply:
Sounds great, will update!